packages/components/src/internal/components/samples/ManageSampleStatusesPanel.tsx

```diff
@@ -402,23 +404,26 @@ SampleStatusesList.displayName = 'SampleStatusesList';
 
 interface ManageSampleStatusesPanelProps extends InjectedRouteLeaveProps {
     api?: ComponentsAPIWrapper;
-    container?: Container;
+    projectContainer?: Container;
```

Comment:
I think this renaming is slightly inaccurate. In the case where you have an LKSM folder under a non-LKSM project, this container will be the folder not the project.

Reply:
Changed to`homeContainer` instead.